doc/LocalVMSetup.md

```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
-Please refer to: https://fedoraproject.org/wiki/EPEL
+## 4. Run the following commands.
```

Comment:
Please explain what these commands do. Do they add EPEL (like it was mentioned before)? If so, please mention it.

Reply:
Yes, please keep this title unchanged:
```
You may need to add EPEL repo.
```
This tells the user what to do next.

Reply:
Please don't change this title here.

It is referenced from "cortx-motr/doc/Quick-Start-Guide.rst"